R/read_bed.r

```diff
@@ -26,9 +26,11 @@
 #' read_bed(valr_example('3fields.bed.gz'), sort = FALSE)
 #'
 #' @export
-read_bed <- function(filename, n_fields = 3, col_types = bed12_coltypes,
+read_bed <- function(filename, n_fields = NULL, col_types = bed12_coltypes,
                      sort = TRUE, ...) {
 
+  if (missing(n_fields)) n_fields <- ncol(read.table(filename, nrows = 5,sep="\t"))
```

Comment:
space after comma.

Looks good, but it would be good to add a message that says i.e. 'n_fields set automatically to 5'. Good excuse to use the `glue` package, which is nicer that ``paste`.

Should also update the `@param` for `n_fields` to say something about auto-detecting field numbers.